src/main/java/org/c4sg/service/impl/OrganizationServiceImpl.java

```diff
@@ -109,18 +110,11 @@ public OrganizationDTO updateOrganization(int id, OrganizationDTO organizationDT
     public void deleteOrganization(int id){
     	Organization organization = organizationDAO.findOne(id);
     	if(organization != null){
-    		organization.setStatus(Constants.ORGANIZATION_STATUS_CLOSED);
-    		organization.setLogoUrl(null);
-    		organizationDAO.save(organization);
-    		List<ProjectDTO> projects=projectService.findByOrganization(id);
-    		for (ProjectDTO project:projects){
-    			projectService.deleteProject(project.getId());
-    		}
-    		organizationDAO.deleteUserOrganizations(id);
     		//TODO: Local or Timezone?
     		//TODO: Format date
     		//organization.setDeleteTime(LocalDateTime.now().toString());
     		//organization.setDeleteBy(user.getUsername());
+    		organizationDAO.save(organization);
```

Comment:
Why are you calling `organizationDAO.save(organization)` method if the method is called `deleteOrganization(int i)` ?